packages/mcl/src/src/mcl/commands/host_info.d

```diff
@@ -34,10 +35,12 @@ struct Params
 {
     // @optional()
     // InfoFormat format = InfoFormat.JSON;
+    @optional() string codaApiToken;
     void setup()
     {
     }
 }
+Params params;
```

Comment:
Avoid global variables, as they make the control flow harder to reason about as the codebase evolves.